README.md

```diff
@@ -367,4 +368,48 @@ flag.
 
 ```
 dbt test --select stg_customers --store-failures
-```
\ No newline at end of file
+```
+
+## compare_which_columns_differ ([source](macros/compare_which_columns_differ.sql))
+This macro generates SQL that can be used to detect which common columns between two relations
+contain any value level changes. It does not return the magnitude of the change, only whether or not a difference has occurred.
+This can be useful when comparing two versions of a model between development and production environments.
+
+
+```sql
+
+{% set prod_relation=adapter.get_relation(
+      database=target.database,
+      schema="prod_schema",
+      identifier="fct_orders"
+) -%}
+
+{% set dev_relation=ref('fct_orders') %}
+
+{{ audit_helper.compare_which_columns_differ(
+    a_relation=prod_relation,
+    b_relation=dev_relation,
+    primary_key="order_id",
+    exclude_columns=["tax_amount"]
+) }}
+
+
+Results:
+
+
+| column_name | has_changed |
```

Comment:
"has_changed" feels vague to me - could we change this to something like "values_differ"